src/migrations/20190809122728-create-sessions-table.ts

```diff
@@ -0,0 +1,46 @@
+import { DataTypes, QueryInterface } from "sequelize";
+
+export function up(queryInterface: QueryInterface): Promise<void> {
+  return queryInterface.createTable("Sessions", {
+    createdAt: {
+      allowNull: false,
+      type: new DataTypes.DATE()
+    },
+    deletedAt: {
```

Comment:
When the user performs a logout the related session is not deleted, instead the attribute `deletedAt` is set. This happens because of the `paranoid: true` setting in the Session model, in order to keep the history of all the sessions created by a user, and that’s the reason why there is no process periodically deleting the expired sessions.
Both the attributes `expirationTime` and `deletedAt` represent the moment since the token is not valid anymore, but the first one is its “natural” expiration time, the latter is the time when the user logged out.